.github/workflows/scripts/linux/appimage.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -ex
```

Comment:
Parameters shouldn't be passed to shebang. See here: https://github.com/PCSX2/pcsx2/pull/4427#discussion_r628949877
```suggestion
#!/bin/bash

set -ex
```